frontend/packages/knative-plugin/src/components/traffic-splitting/TrafficSplittingModal.tsx

```diff
@@ -10,7 +10,7 @@ import {
   MultiColumnField,
   InputField,
   DropdownField,
-} from '@console/dev-console/src/components/formik-fields';
+} from '@console/shared/src/components/formik-fields';
```

Comment:
Oh, this is a great change. Good catch! `knative-plugin` shouldn't rely on `dev-console`.